hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/HoodieFileGroupReaderBasedFileFormat.scala

```diff
@@ -61,20 +64,21 @@ trait HoodieFormatTrait {
 
 /**
  * This class utilizes {@link HoodieFileGroupReader} and its related classes to support reading
- * from Parquet formatted base files and their log files.
+ * from Parquet or ORC formatted base files and their log files.
  */
```

Comment:
I'm generally ok with this, but there is a method that is returning that it is a parquet format, and if you change it to something else then there will be optimizations that don't get used. So just be advised

Reply:
Should I make this class extend FileFormat directly or do you think it makes more sense to have a class for ORC and one for multi-format?